app/models/auction_parser.rb

```diff
@@ -45,8 +45,8 @@ def strong_params
 
   def delivery_due_at
     if due_in_days.present?
-      real_days = due_in_days.to_i.business_days
-      end_of_workday(real_days.after(ended_at))
+      real_days = due_in_days.to_i
+      DefaultDeadlineDateTime.new(ended_at, real_days).dc_time
```

Comment:
idea: let's used named args here rather than depending on ordering. So `DefaultDeadlineDateTime.new(start_time: ended_at, day_offset: real_days).dc_time` --I think that will make this easier to understand